### ISSUE TITLE
Visual glitches while scrolling - Bad performance - Not smooth visualization

### ISSUE THREAD

**Issue by NicolasYankov** [Akryum/vue-virtual-scroller#278]
Content doesn't render smoothly while scrolling on desktop (Chrome/Firefox) and mobile devices. Visual glitches appear even if you scroll very slowly. This happens with both **DynamicScroller** and **RecycleScroller**. In my case I have a list of 300 cards with variable height, approximately 180px each. The card is composed mostly of **Vuetify 2.0** components `<v-row>, <v-col>, <v-flex>, <v-avatar>, <-v-chip>, <v-btn>`, etc., where most of them have `display: flex, justify-content, align-self` and other alignment CSS properties.

### [**Demo Scroll Issue Here**](https://codesandbox.io/s/github/s-a-s-h-o/scroll-performance-issue)

**Example: DynamicScroller pseudo configuration**

```
<DynamicScroller :items="data" :min-item-size="100" :buffer="500" key-field="id">
 <template v-slot="{ item, index, active }">
  <DynamicScrollerItem :item="item" :active="active" :data-index="index">
    <v-card>
      <v-row>
        <!-- Card Content Here  --> 
      </v-row>
   </v-card>
  </DynamicScrollerItem>
 </template>
</DynamicScroller>
```
I ran several performance tests and the overall FPS drops drastically when scrolling. You can see it here -><blockquote class="imgur-embed-pub" lang="en" data-id="a/tzbgpuF"><a href="//imgur.com/a/tzbgpuF">Vue Virtual Scroller - Performance Issues</a></blockquote>

I found out that each DynamicScrollerItem represents a layer (in Chrome Layers tool) which is positioned with the help of transform: 'translateY(number)'. Could this cause the issue? I tried various configurations with min-item-size, buffer and other props with no avail.

Any ideas are welcome and thank you for your attention!

**Comment by danielgindi** [comment]
I tested this, didn't see any issues. Could be a specific mouse/touchpad scrolling issue - some of them (like on the mac) do not fire `scroll` events while smooth scrolling. This is an issue with any virtual list.

**Comment by mech01nc01** [comment]
no Problems here wih you demo, but i've found this because i have a smilar issue with a "flextable"
even with just 21 items it laggs, buti dont't know why

**Comment by rogeralsing** [comment]
I have seen bad rendering performance with the Vuetify components, unrelated to virtual scroller.

e.g. buttons, textboxes are very heavy to render.

**Comment by aalyokhin** [comment]
@nikolay-yankov have you ever been able to resolve this? having the same situation with Vuetify.

**Comment by nikolay-yankov** [comment]
@aalyokhin Nope, the only thing which helped a little bit is to use fewer Vuetify elements as some of its styles increase this effect.

**Comment by patrykharanczyk** [comment]
I had the same issue. For me including below import in my `index.ts` file fixed the issue.
`import 'vue-virtual-scroller/dist/vue-virtual-scroller.css'`
